### PR TITLE
HyperShift QE: enable longduration e2e test

### PR DESCRIPTION
**Pull request by LiangquanLi930**
enable long duration e2e test on HyperShift CI

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 1]
```diff
@@ -1042,6 +1042,20 @@ tests:
...
+- as: aws-ipi-ovn-hypershift-guest-f28-longduration
+  cron: 43 9 25 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@amd64 and @aws-ipi and @network-ovnkubernetes and @hypershift-hosted
+        and not @fips'
+      TAG_VERSION: '@4.14'
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&
```
Comment (LiangquanLi930) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1054]: @kuiwang02 Could help to confirm it?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 2]
```diff
@@ -1079,6 +1093,18 @@ tests:
...
     - chain: openshift-e2e-test-hypershift-qe-mgmt
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift
+- as: aws-ipi-ovn-hypershift-mgmt-f28-longduration
+  cron: 58 13 24 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      TEST_ADDITIONAL: Hypershift|Network_Observability
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~MicroShiftOnly&;HyperShiftMGMT&
```
Comment (LiangquanLi930) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1103]: Same

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml (hunk 1 above)
Comment (kuiwang02) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1054]: it should be `~ChkUpgrade&;NonPreRelease&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&`

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 3]
```diff
@@ -1042,6 +1042,20 @@ tests:
...
+  cron: 43 9 25 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@amd64 and @aws-ipi and @network-ovnkubernetes and @hypershift-hosted
+        and not @fips'
+      TAG_VERSION: '@4.14'
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&
+      TEST_TIMEOUT: "30"
```
Comment (kuiwang02) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1055]: @LiangquanLi930 better to take default value 75, not need to overwrite it. if you want to reset it, better to set it as 90.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml (hunk 2 above)
Comment (kuiwang02) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1103]: it should be `~ChkUpgrade&;NonPreRelease&;~DisconnectedOnly&;HyperShiftMGMT&;~MicroShiftOnly&`

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 4]
```diff
@@ -1079,6 +1093,18 @@ tests:
...
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift
+- as: aws-ipi-ovn-hypershift-mgmt-f28-longduration
+  cron: 58 13 24 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      TEST_ADDITIONAL: Hypershift|Network_Observability
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~MicroShiftOnly&;HyperShiftMGMT&
+      TEST_TIMEOUT: "60"
```
Comment (kuiwang02) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1104]: @LiangquanLi930 same to above.

**Comment by LiangquanLi930** [comment]
@liangxia Could you help to review this PR? Thanks

**Comment by kuiwang02** [comment]
i am ok from golang's view.

File: ci-operator/step-registry/openshift/e2e/test/hypershift-qe/mgmt/longduration/openshift-e2e-test-hypershift-qe-mgmt-longduration-chain.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+chain:
+  as: openshift-e2e-test-hypershift-qe-mgmt-longduration
+  steps:
+  - ref: cucushift-hypershift-extended-health-check
+  - ref: idp-htpasswd
+  - ref: openshift-extended-test-longduration
+  documentation: |-
+    Execute e2e tests from QE, which include golang (openshift-extended-test), cucushift (cucushift-e2e), ...(more to add)
```
Comment (liangxia) [inline ci-operator/step-registry/openshift/e2e/test/hypershift-qe/mgmt/longduration/openshift-e2e-test-hypershift-qe-mgmt-longduration-chain.yaml:8]: The documentation needs update.

Comment (LiangquanLi930) [reply ci-operator/step-registry/openshift/e2e/test/hypershift-qe/mgmt/longduration/openshift-e2e-test-hypershift-qe-mgmt-longduration-chain.yaml]: @liangxia Thanks a lot, Could you help to review again?

**Pull request merged**